rllab/misc/logger.py

```diff
@@ -1,12 +1,17 @@
-from enum import Enum
-
 from rllab.misc.tabulate import tabulate
-from rllab.misc.console import mkdir_p, colorize
+from rllab.misc.console import mkdir_p
+from rllab.misc.console import colorize
 from rllab.misc.autoargs import get_all_parameters
+from rllab.misc.tensorboard_summary import Summary
+
+import tensorflow as tf
+
 from contextlib import contextmanager
+from enum import Enum
 import numpy as np
-import os
+
 import os.path as osp
+import os
```

Comment:
Please reorder imports to conform to PEP8.

https://www.python.org/dev/peps/pep-0008/#imports